helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -300,6 +300,9 @@ public void addJob(final Job job) throws JobExistsException {
         }
         throw new JobExistsException(id.toString());
       }
+    } catch (NoNodeException e) {
```

Comment:
If the /config or /config/jobs paths haven't been created for some reason then the `create` ops in the transaction above will throw.